src/sentry/workflow_engine/models/detector.py

```diff
@@ -15,6 +15,13 @@ class Detector(DefaultFieldsModel, OwnerModel):
     name = models.CharField(max_length=200)
     data_sources = models.ManyToManyField("workflow_engine.DataSource", through=DataSourceDetector)
 
+    condition_group = FlexibleForeignKey(
```

Comment:
Wondering if this should be named like `workflow_condition_group` or maybe have a comment explaining how it's used?